tests/unit/ui/components/TestUserCollection.js

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+
+import UserCollection, {
+  UserCollectionBase,
+} from 'ui/components/UserCollection';
+import { fakeI18n, shallowUntilTarget } from 'tests/unit/helpers';
+import LoadingText from 'ui/components/LoadingText';
+
+function render(customProps = {}) {
```

Comment:
We could probably move this function into the top-level `describe` block, what do you think?